roles/keystone-setup/tasks/main.yml

```diff
@@ -42,6 +42,7 @@
                  tenant={{ item.tenant }}
                  token={{ admin_token }}
   with_items: keystone.users
+  when: "{{ item.name }}.enabled is undefined or {{item.name}}.enabled|bool"
```

Comment:
I removed this change since it didn't make sense since openstack users are not (currently) associated with services. 

The error that I was seeing was with keystonev3 or cinderv2 or heat-cfn. Since there is no dict item defined for either of those, ansible erred out with unknown error.